rfcs/0146-meta-categories.md

```diff
@@ -264,11 +264,15 @@ Still unsolved is what data structure is better suited to represent a category.
 # Future work
 [future]: #future-work
 
-- Create a team to manage categorization-related issues, including but not
-  limited to:
-  - Curate the categories.
-  - Update documentation.
-  - Update Continuous Integration.
+Given the typical complexities that arise from categorization, and expecting
+that regular maintainers are not expected to understand its minuteness, it is
+strongly recommended the creation of a team entrusted to manage issues related
+to categorization, including but not limited to:
```

Comment:
I want to be clear that some decisions by a team carry the authority granted by an RFC. So, a bit of RFC46-style, work→authority style proposal:

… it is strongly recommended to create a team that would take up the work of maintaining and curating the categorisation, as well as the corresponding documentation. A team doing this work will automatically receive the additional authority to:

- Resolve the disputes over categorisation of specific packages.
- Decide when a Continuous Integration check for categorisation is ready to be developed with «gray/neutral» failure statuses, and when a CI check with a good track record in «gray» mode can be upgraded to «red/blocking» failures.
- Coordinate the treewide efforts to import or update the categorisation of packages.